releasenotes/notes/add-bit-register-rust-979dd113aefb3563.yaml

```diff
@@ -0,0 +1,23 @@
+---
+features_circuits:
+  - |
+   Python classes :class:`.Bit` and :class:`.Register` have been ported to Rust
+   along with all their respective subclasses including:
+   
+   * :class:`.Qubit` and :class:`.QuantumRegister`.
+   * :class:`.Clbit` and :class:`.ClassicalRegister`.
+   * :class:`.AncillaQubit` and :class:`.AncillaRegister`.
```

Comment:
What's the user impact for the feature here? Is it faster, etc? The rust refactor is not really a user facing change because the interface the users consume hasn't changed. We normally reserve feature notes for user facing features.

Reply:
No user inpact whatsoever, so then should I just remove that section and just mention the upgrade parts?

Reply:
Yeah, if there isn't a user impact I would just remove this from the release notes

Reply:
Done in 0e669440a.